clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ClusterController.groovy

```diff
@@ -200,8 +192,11 @@ class ClusterController {
       throw new NotFoundException("Target not found (target: ${target})")
     }
 
+    // we can optimize loads iff the cloud provider supports loading minimal clusters (ie. w/o instances)
+    def shouldExpand = !clusterProviders.find { it.cloudProviderId == cloudProvider }.supportsMinimalClusters()
+
     // load all server groups w/o instance details (this is reasonably efficient)
-    def sortedServerGroups = getServerGroups(application, account, clusterName, cloudProvider, null /* region */, false).findAll {
+    def sortedServerGroups = getServerGroups(application, account, clusterName, cloudProvider, null /* region */, shouldExpand).findAll {
```

Comment:
Maybe I'm misreading this, but for the non-aws case `supportsMinimalClusters` is false, so `shouldExpand` is true. The old default was passing in false, but now it's true. Is that expected?

Reply:
The pre-perf change default was true (or more specifically it wasn't defined and was true inside each cloud provider)

Reply:
With the change, *only* aws should have `shouldExpand` = false.

Reply:
Ok sounds good